src/lsst/rsp/startup/services/labrunner/labrunner.py

```diff
@@ -783,21 +778,114 @@ def _make_abnormal_landing_page(self) -> None:
             settings.parent.mkdir(exist_ok=True, parents=True)
             settings.write_text(s_txt)
         except Exception:
-            self._logger.exception("Writing abnormal startup files failed")
+            self._logger.exception(
+                "Writing files to report abnormal startup failed"
+            )
+
+    def _make_abnormal_landing_markdown(self) -> str:
+        user = self._env["USER"]
+        home = self._env.get(
+            "NUBLADO_HOME",
+            self._env.get(
+                "HOME",
+                f"/home/{user}",  # Guess, albeit a good one.
+            ),
+        )
+
+        errmsg = self._env.get("ABNORMAL_STARTUP_MESSAGE", "<no message>")
+        errcode = self._env.get("ABNORMAL_STARTUP_ERRORCODE", "EUNKNOWN")
+
+        self._logger.error(
+            f"Abnormal startup: errorcode {errcode}; message {errmsg}"
+        )
+
+        open_an_issue = dedent(
+            f"""
+
+            Please open an issue with your RSP site administrator with the
+            following information: `{errmsg}`
+            """
+        )
+
+        # Start with generic error text.  It's very simple markdown, with a
+        # heading and literal text only.
+
+        txt = dedent("""
+        # Abnormal startup
+
+        Your Lab container did not start normally.
+
+        Do not trust this lab for work you want to keep.
+
+        """)
+
+        # Now add error-specific advice.
+        match errcode:
+            case "EDQUOT":
+                txt += dedent(
+                    f"""
+                    You have exceeded your quota.  Try using the terminal to
+                    remove unneeded files in `{home}`.  You can use the
+                    `quota` command to check your usage.
+
+                    After that, shut down and restart the lab.  If that does
+                    not result in a working lab:
+                    """
+                )
+            case "ENOSPC":
+                txt += dedent(
+                    f"""
+                    You have run out of filesystem space.  Try using the
+                    terminal to remove unneeded files in `{home}`.  Since the
+                    filesystem is full, this may not be something you can
+                    correct.
+
+                    After you have trimmed whatever possible, shut down and
+                    restart the lab.
+
+                    If that does not result in a working lab:
```

Comment:
I'm having a bit of trouble following from the PR view, but what content comes after this?

Reply:
```
        open_an_issue = dedent(
            f"""

            Please open an issue with your RSP site administrator with the
            following information: `{errmsg}`
            """
        )
```

Reply:
This is more of the "we're going to give you something to paste into the CLO post you will make saying this doesn't work" stuff.

One thing that I don't know what's going to happen: are users more likely to read the markdown page after they click away the dialog box?  Or are they more likely to copy stuff out of the dialog box and never even notice the markdown page?

Reply:
In the interests of getting this merged,per @frossie , I'm closing it.  However, for the record, the suggestions are very good and we should implement at least most of them sometime.